DebuggerGame/Assets/Scripts/Board Scripts/Board.cs

```diff
@@ -179,8 +178,7 @@ private void Start()
                         => board != null && boundsEnabled
```

Comment:
Collidables being enabled should be separate from bounds being enabled, so there should probably be a separate variable